pentaho-gwt-widgets/build-res/subfloor.xml

```diff
@@ -209,7 +209,7 @@ SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
             description="Unit test html reports are placed here"/>
   <property name="junit.haltonfailure" value="no" description="Fail the build if a test fails"/>
   <property name="junit.haltonerror" value="no" description="Fail the build if a error occurs"/>
-  <property name="junit.maxmemory" value="256M" description="Heap size when Junit is run in fork mode"/>
+  <property name="junit.maxmemory" value="1024M" description="Heap size when Junit is run in fork mode"/>
```

Comment:
We do not modify subfloor directly. Put this override in the local build file please.